docs/source/accelerate_training.mdx

```diff
@@ -0,0 +1,91 @@
+<!---
+Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Accelerating Training
+
+Gaudi offers several possibilities to make training faster.
+They are all compatible with each other and can be coupled with [distributed training](distributed).
+
+
+## Lazy Mode
+
+Two execution modes are proposed:
+- *Lazy mode*, where the Habana bridge internally accumulates operations in a graph. The execution of the operations in the accumulated graph is triggered in a lazy manner. This allows the bridge to construct a graph with multiple operations, which provides the graph compiler the opportunity to optimize the device execution for these operations.
+- *Eager mode*, where one operation at a time is executed.
+
+In lazy mode, the graph compiler generates optimized binary code that implements the given model topology on Gaudi. It performs operator fusion, data layout management, parallelization, pipelining and memory management, as well as graph-level optimizations.
+
+To execute your training in lazy mode, you must provide the following training arguments:
+```python
+args = GaudiTrainingArguments(
+    # same arguments as in Transformers,
+    use_habana=True,
+    use_lazy_mode=True,
+    gaudi_config_name=path_to_my_gaudi_config
+)
+```
+
+
+## Mixed-Precision Training
+
+Mixed-precision training enables to compute some operations using lighter data types to accelerate training.
+Habana Mixed Preicision (HMP) proposes to mix *fp32* and *bf16* operations.
+In order to do so, you must set `"use_habana_mixed_precision"` to `true` and `"hmp_opt_level"` to `"O1"` in the Gaudi configuration file.
+Then, you can specify which operators to compute in *bf16* with `"hmp_bf16_ops"` and which operators to compute in *fp32* with `"hmp_fp32_ops"`.
+If these operators are not specified, their default values are set to be the ones written in the [Gaudi configuration file of BERT](https://huggingface.co/Habana/bert-large-uncased-whole-word-masking/blob/main/gaudi_config.json), which is a good starting point for applying HMP:
```

Comment:
did you have a default list for each model? our default list is Default BF16 list = [addmm, bmm, conv1d, conv2d, conv3d, dot, mm, mv]

Default FP32 list = [batch_norm, cross_entropy, log_softmax, softmax, nll_loss, topk]
see https://docs.habana.ai/en/latest/PyTorch/PyTorch_User_Guide/PT_Mixed_Precision.html#opt-level-o1

Reply:
Yes, the default lists are defined in `gaudi_configuration.py`, you can see them [here](https://github.com/huggingface/optimum-habana/blob/main/optimum/habana/gaudi_configuration.py). Those are basically the list of ops we use for BERT/DistilBERT/RoBERTa/ALBERT.

You want me to modify them so that they are the same as in Synapse?

Reply:
no, what you have is fine since it's for transformer related.